css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  padding: 13px;
+  font-size: 20px;
+}
+
+button:hover {
+  background-color: rgb(22, 22, 22);
+  color: rgb(224, 224, 224);
+}
```

Comment:
You still need to change the cursor to a pointer

Reply:
Does it matter whether you add the cursor to the regular element or the psuedo element?

Reply:
Either way is totally fine!